agave_app/aboutDialog.cpp

```diff
@@ -0,0 +1,94 @@
+#include "aboutDialog.h"
+
+#include <QApplication>
+#include <QDialogButtonBox>
+#include <QLabel>
+#include <QTextBrowser>
+#include <QVBoxLayout>
+
+AboutDialog::AboutDialog()
+  : QDialog()
+{
+  setWindowTitle("About AGAVE");
+  auto layout = new QVBoxLayout(this);
+  auto label = new QLabel(this);
+
+  label->setText("<h1><b>AGAVE</b></h1>"
+                 "<i>Advanced GPU Accelerated Volume Explorer</i>"
+                 "<h4><b>Version " +
+                 qApp->applicationVersion() +
+                 "</b></h4>"
+                 "");
+  label->setAlignment(Qt::AlignCenter);
+  layout->addWidget(label);
+
+  label = new QLabel(this);
+  label->setText(
+    "<a href=\"https://forum.image.sc/tag/agave\">Support forum</a><br/>"
+    "<a href=\"https://allen-cell-animated.github.io/agave\">Online documentation</a><br/>"
+    "<a href=\"https://github.com/allen-cell-animated/agave/issues\">Report a bug or make a request</a><br/>"
+    "<a href=\"https://github.com/allen-cell-animated/agave\">Source code</a><br/>"
+    "");
```

Comment:
Nit: If we ever add more links it might be nice to use a shared constants file across this and the other actions down below.

Reply:
removed from "about" dialog